scripts/ldmx-recompileAndFire.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+############################# Usage #############################
+# ldmx ./scripts/ldmx-recompileAndFire.sh
+# for changing the defaults
+# ldmx setenv LDMX_COMPILE_CORES=<num cores to be used>, i.e.
+# ldmx setenv LDMX_COMPILE_CORES=16
+# or
+# ldmx setenv LDMX_COMPILE_BUILD=<build location>
+
+FIREINPUT=$1
```

Comment:
Instead of storing only the first argument, we should pass the entire command line to `fire`. For this reason, I'm just removing this local variable and giving `$@` to `fire` below.

```suggestion
```